.github/workflows/build_packages.yaml

```diff
@@ -77,7 +77,10 @@ jobs:
           echo "VERSION=$(./pkg-vsn.sh $PROFILE)" >> $GITHUB_OUTPUT
       - name: get_all_deps
         run: |
-          make -C source deps-all
+          pusd source
+          make clean-all
+          make deps-emqx deps-emqx-enterprise
```

Comment:
Since emqx-enterprise is a superset of emqx, maybe only `deps-emqx-enterprise` would be enough?